homeassistant/components/xiaomi_miio/number.py

```diff
@@ -244,6 +273,15 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
                 description.max_value = OSCILLATION_ANGLE_VALUES[model].max_value
                 description.min_value = OSCILLATION_ANGLE_VALUES[model].min_value
                 description.step = OSCILLATION_ANGLE_VALUES[model].step
+            if (
+                description.key == ATTR_LED_BRIGHTNESS_LEVEL
+                and model in LED_BRIGHTNESS_VALUES
+            ):
+                description.key = LED_BRIGHTNESS_VALUES[model].key
```

Comment:
I notice now that we are modifying the constant description here. That's not good since the constant is not expected to be mutated. I think we should solve this some other way.

Reply:
OK, I'll prepare a different solution.